packages/expo-keep-awake/ios/EXKeepAwake/EXKeepAwake.m

```diff
@@ -13,7 +13,13 @@ @interface EXKeepAwake () <UMAppLifecycleListener>
 @end
 
 @implementation EXKeepAwake {
-  BOOL _active;
+  NSMutableSet *_activeTags;
+}
+
+- (instancetype)init {
+  self = [super init];
```

Comment:
A typical Objective-C class initializer looks like:
```objc
if (self = [super init]) { // or other super initializer
  // initialization of this class properties
}
return self;
```

It may happen that `[super init]` will return `nil`, in which case we can't assign to `self.` properties. https://stackoverflow.com/questions/1287950/in-objective-c-why-should-i-check-if-self-super-init-is-not-nil, http://www.cocoawithlove.com/2009/04/what-does-it-mean-when-you-assign-super.html 🙂